inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java

```diff
@@ -195,6 +195,9 @@ public static KafkaExtractNode createExtractNode(KafkaSource kafkaSource) {
             default:
                 startupMode = KafkaScanStartupMode.LATEST_OFFSET;
         }
+        if (StringUtils.isNotBlank(kafkaSource.getGroupId())) {
+            startupMode = KafkaScanStartupMode.GROUP_OFFSETS;
+        }
```

Comment:
Why  to set startupMode by judging groupid? `startupMode` should be setted by front.

Reply:
Because when enable group offset, the `startUpmode` should be "group-offsets", and group id must be set.

Reply:
> Because when enable group offset, the `startUpmode` should be "group-offsets", and group id must be set.

Consumer group is not relative with `startUpMode`. I aslo should set consumer group when use other startupmode.

Reply:
Yes, I misunderstood. Fixed.